cmd/file_convert.go

```diff
@@ -37,7 +40,13 @@ func executeConvert(_ *cobra.Command, _ []string) error {
 			return nil
 		}
 
-		err = convert.Convert(convertCmdInputFile, convertCmdOutputFile, sourceFormat, destinationFormat)
+		err = convert.Convert(
+			[]string{convertCmdInputFile},
+			convertCmdOutputFile,
+			file.Format(strings.ToUpper(convertCmdStateFormat)),
```

Comment:
what happens if the user uses `yml` instead of `yaml` does it panic? shouldn't we validate this input?

Reply:
That value is simply passed along all the way to writing the final file, where it gets validated;  https://github.com/Kong/deck/blob/main/file/writer.go#L737-L749

Reply:
but `file.Format(...)` would pass?